.github/workflows/release_parallel.yml

```diff
@@ -0,0 +1,218 @@
+name: Parallel Release
+on:
+    workflow_call:
+        secrets:
+            GPG_SIGNING_KEY:
+                required: true
+            GH_TOKEN:
+                required: true
+            NPM_TOKEN:
+                required: true
+            LASTMJS_GITHUB_TOKEN:
+                required: true
+
+jobs:
+    prepare-release:
+        name: Prepare Release
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }} # All commits must be verified
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        outputs:
+            release-version: ${{ steps.get-version.outputs.release-version }}
+            test-infos: ${{ steps.get-test-infos.outputs.test-infos }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.event.pull_request.head.ref || github.ref }}
+                  token: ${{ secrets.LASTMJS_GITHUB_TOKEN || github.token }}
```

Comment:
What's with the or? Not sure we want to not explicitly use the lastmjs token like this

Reply:
I don't know, it has been that way from time immemorial, dating back to at least 2 years ago when Dan first ran prettier on our code base

Reply:
open an issue to explore this for 1.0 milestone

Reply:
https://github.com/demergent-labs/azle/issues/2200